libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Input/InputDialog.cs

```diff
@@ -201,6 +219,14 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
             }
         }
 
+        /// <summary>
+        /// Called when the dialog is _continued_, where it is the active dialog and the
+        /// user replies with a new activity.
```

Comment:
Same as in PR 331 https://github.com/southworks/botbuilder-dotnet/pull/331#discussion_r484559583